### DISCUSSION TITLE
Autofocus on TextField

### DISCUSSION THREAD

**Discussion by mperrando** [buildo/bento-design-system#286] [Q&A] [answered]
Hi!
How can I put focus on a TextField on component mount?

I tried
* with the "autofocus" property, but it is missing
* through a ref, but TextField has no ref

I am short on ideas.

Thank you.

**Comment by gabro** [comment, marked as answer]
Good point, we currently do not offer a facility for auto focus for fields, but I agree it‘d make sense we would. We’ll schedule this soon. Fn the meantime you could work around this by using something like https://react-spectrum.adobe.com/react-aria/FocusScope.html, and wrapping the `TextField` with a `<FocusScope autoFocus>`

**Reply by gabro** [reply]
@mperrando this has now been released in 0.12.2
